<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Vocabs">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>541</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
  font-size: 24px;
font-color : #000000;
        background-color: #f0f0f0;
        padding: 10px;
        border-radius: 25px;
        position: absolute;
        top: 50%;
        left: 50%;
        transform: translate(-50%, -50%);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Random_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">  font-size: 18px;
        background-color: #f0f0f0;
        padding: 10px;
        border-radius: 25px;
        position: absolute;
        top: 50%;
        left: 50%;
        transform: translate(-50%, -50%);</string>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainKanji">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>541</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
  font-size: 40px;
font-color : #000000;
        background-color: #f0f0f0;
        padding: 10px;
        border-radius: 25px;
        position: absolute;
        top: 50%;
        left: 50%;
        transform: translate(-50%, -50%);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="meaning_title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Meaning</string>
   </property>
  </widget>
  <widget class="QLabel" name="yomis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="see_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">  font-size: 14px;
        background-color: #f0f0f0;
        padding: 10px;
        border-radius: 25px;
        position: absolute;
        top: 50%;
        left: 50%;
        transform: translate(-50%, -50%);</string>
   </property>
   <property name="text">
    <string>See the Meanings</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="grade">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>161</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Grade</string>
   </property>
  </widget>
  <widget class="QLabel" name="stroke">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Historic</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-20</y>
     <width>641</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Downloads/clay-banks-u27Rrbs9Dwc-unsplash (1).jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>Random_btn</zorder>
  <zorder>meaning_title</zorder>
  <zorder>yomis</zorder>
  <zorder>see_btn</zorder>
  <zorder>stroke</zorder>
  <zorder>grade</zorder>
  <zorder>label</zorder>
  <zorder>mainKanji</zorder>
  <zorder>Vocabs</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
